protobuf/module_msg.proto

```diff
@@ -652,10 +652,13 @@ message MACSwapArg {
  */
 message MeasureArg {
   // int64 warmup = 1; /// removed: instead of warmup delay, user should Clear()
-  uint64 offset = 2; /// Where to store the current time within the packet, offset in bytes.
-  double jitter_sample_prob = 3; /// How often the module should sample packets for inter-packet arrival measurements (to measure jitter).
-  uint64 latency_ns_max = 4; /// maximum latency expected, in ns (default 0.1 s)
-  uint32 latency_ns_resolution = 5; /// resolution, in ns (default 100)
+  oneof type {
+    uint64 offset = 2; /// Where to store the current time within the packet, offset in bytes.
+    string attr_name = 3; /// Where to store the current time as attribute
+  }
```

Comment:
We have not been good about this in the past, so this is just a suggestion: when changing a protobuf like this, try not to renumber the existing fields. That is, give `attr_name` the index 6 (instead of 3) and leave the others alone. That way old controllers that do not have `attr_name` built in to them will interact well with a new BESS that uses `attr_name` and the other fields.

(This does not affect `TimestampArg` since there's only the one field so far.)